pandas/core/groupby/ops.py

```diff
@@ -525,6 +525,15 @@ def _call_cython_op(
                     result_mask=result_mask,
                     is_datetimelike=is_datetimelike,
                 )
+            elif self.how == "mean":
```

Comment:
we don't want to add yet another path here. so prob best to just add kwargs to the other aggregation functions (ex min/max) and passthru is_datetimelike

Reply:
Done.